src/modules/endpoint/sagas/loadCategories.js

```diff
@@ -22,8 +22,10 @@ function * loadCategories (
   dataContainer: DataContainer,
   shouldUpdate: boolean
 ): Saga<FetchMapType> {
-  const context = new DatabaseContext(city, language)
-  const categoriesAvailable = yield call({ context: dataContainer, fn: dataContainer.categoriesAvailable }, context)
+  const categoriesAvailable = yield call(
+    (city: string, language: string) => dataContainer.categoriesAvailable(city, language),
```

Comment:
Wouldn't this work?
```suggestion
   yield call(dataContainer.categoriesAvailable, city, language)
```

Reply:
No, because categoriesAvailable(...) references `this`

Reply:
Shouldn't we just use arrow functions then? Maybe we should find a rule of thumb when to use which (for the sake of consistency)

Reply:
It depends on the function you are calling. If you call a pure and static function you do not need lambdas. If you are calling a nonpure function on an instance you need a lambda or that other syntax.